src/run/run.c

```diff
@@ -2523,10 +2528,6 @@ static int start_transient_scope(sd_bus *bus) {
                 }
         }
 
-        /* Stop agents before we pass control away, to avoid TTY conflicts */
-        polkit_agent_close();
```

Comment:
One of those days where you'd really wish the pidfd kill-on-close thing is ready